docs/direct/api/v1.md

```diff
@@ -1485,490 +1862,2635 @@ The properties of each endpoint type are listed below.
+
+```
+DELETE /customers/{customerId}/linkedusers/{linkeduserId}?removeUser=true
+```
+
+There are 2 kinds of user: Web User (which is a single user-login) and Linked User (which is an association between Web User and an Account). So you can have a user on multiple accounts - e.g an IT Technician who supports 3 schools with their own accounts may have a single Web User but then will have Linked Users across 3 separate accounts.<br />
+With the above in mind, when you **DELETE** a Linked User you only delete that association rather than the Web User too. You can still link the existing Web User to a new account in this scenario if you wish.
+However, if you'd like to fully remove a user, when you submit your Linked User delete, add in an additional query parameter of *removeUser* with a value of true: "removeUser": true
+
+## Charging Plans
+
+| Resource Information           |                                      |
+|:-------------------------------|:-------------------------------------|
+| Location                       | /chargingPlans
+| Rate Limited?                  | Yes                                  |
+| Requests per rate limit window | 1200/hour                            |
+| Authentication                 | Basic Auth                           |
+| HTTP Method(s)                 | [GET](#get) (PUT/POST/DELETE) - not allowed |
+| JSON Representation(s)         | chargingplans                      |
+| Additional Formats             | None                                 |
+
+The Available Charging Plans resource returns a list of charging plans that can be used, represented by a `chargingplans` instance.
+This resource is used to set customers price plans. 
+
+
+
+```json exampleRepresentation
+ {
+  "type": "chargingplan",
+  "id": "1",
+  "uri": "https://pbx.sipcentric.com/api/v1/chargingplans/1",
+  "name": "Standard",
+  "currency": "GBP",
+  "domainCostMinute": 0.000,
+  "roundSeconds": 1,
+  "minDuration": 20,
+  "minCharge": 0.000
+}
+```
+
+
+
+| Type: *chargingplans*           |                                                       |
+|:-------------------------------|:------------------------------------------------------|
+| **Property**                   | **Description**                                       |
+| type <br /> *required*        | The resource type ("chargingplans") <br /> *String*    |
+| id <br /> *read-only*          | The resource chargingplans id<br /> *String*               |
+| uri <br /> *read-only*        | URI to get this chargingplan. <br /> *String*                 |
+| currency <br /> *required*     | The currency for this charging plan("GBP") <br /> *date*        |
+| domainCostMinute <br /> *required* |  <br /> *float* |
+| roundSeconds <br /> *requried*   |  <br /> *integer*                 |
+| minDuration <br /> *read-only*     | minimum duration <br /> *integer*                 |
+| minCharge <br /> *read-only*     | what is the charge per minute  <br /> *float*    |
```

Comment:
Formatting, capitalisation.

Reply:
Also, minCharge is the minimum call charge. Resolved